spec/controllers/my_controller_spec.rb

```diff
@@ -345,5 +345,29 @@
         end
       end
     end
+
+    describe 'file storage' do
+      let(:client) { create(:oauth_client, integration: create(:nextcloud_storage)) }
+      let(:token) { create(:oauth_client_token, oauth_client: client, scope: nil, user:, expires_in: 3_600) }
+
+      render_views
+
+      before { token }
+
+      it 'list the tokens' do
+        get :access_token
+        expect(response.body).to have_selector("#storage-oauth-token-#{token.id}")
+      end
+
+      it 'can remove the token' do
+        expect do
+          delete :delete_storage_token, params: { id: token.id }
+        end.to change(OAuthClientToken, :count).by(-1)
+
+        expect(flash[:info]).to be_present
+        expect(flash[:error]).not_to be_present
+        expect(response).to redirect_to(action: :access_token)
+      end
+    end
```

Comment:
Nice one 👍🏾 
🟢  Noticed we don't have a feature spec, do you think it's worth repurposing into a feature spec instead? I reckon it simulates a more accurate user journey

Reply:
Everything we can test in unit test I'd rather test in unit tests. Feature specs do have a much bigger ramp up time.

So, I'd say, unless we encounter a nasty regression we couldn't test in a unit test, we should bother with a feature specs in this view.

Reply:
@akabiru I thought about it, but seemed so minor that the overhead of the feature spec would be way to high for such a small piece of code.

@Kharonus agreed.